prep-html-script-tag/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Script Tag</title>
+  </head>
+  <body>
+    <script> console.log("I have added JS to a script tag in the HTML!"); </script>
+    <script src="main.js"></script>
+    <script> console.log("I have added a script file via the src attribute!"); </script>
```

Comment:
make sure you put this console log in the `main.js` file, then the code will run from there